articles/css-grid/index.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/css-flexbox/
+title: Create Two-Dimensional Layouts for Websites using CSS Grid
+description: CSS Grid is a two-dimensional positioning layout system in CSS, that can be used to create responsive interfaces for the web.
+author: saiharsha-balasubramaniam
+date: 2020-09-16T00:00:00-08:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/css-grid/hero.jpg
+    alt: css grid
+---
+
+CSS Grid is a two-dimensional positioning layout system in CSS, that can be used to create responsive interfaces for the web. Similar to flexbox, CSS rules are applied to the parent container and the children.
+
+CSS Flexbox is a one dimension positioning system. Check out [this](https://www.section.io/engineering-education/css-flexbox/#conclusion) article to learn more about CSS Flexbox.
+
+## Table of Contents
+
+- [Introduction & History](#introduction-&-history)
+- [Terminology](#terminology)
+- [Let's Code](#let's-code)
+- [Further Reading](#further-reading)
+
+## Introduction & History
+
+Using CSS for laying out items on a webpage has always been a tricky affair. We used tables and floats to position items before. They were inefficient hacks and they weren't intuitive.
+
+Therefore, CSS Flexbox was introduced and it made positioning easier. But as the complexity of your website layout increased, flexbox wasn't the best option. It was designed for one-dimensional website layouts. Thus, CSS grid solves that problem by introducing features for building two-dimensional layouts.
+
+Let us look into some basic terminology used in grid and dive into building a layout using the CSS grid system.
+
+## Terminology
+
+### grid container
+
+This container acts as the parent of all the grid elements.
+
+```html
+<div class="grid-parent-container">
+  <div class="grid-child child-1"></div>
+  <div class="grid-child child-2"></div>
+  <div class="grid-child child-3"></div>
+  <div class="grid-child child-4"></div>
+  <div class="grid-child child-5"></div>
+</div>
```

Comment:
You should add a screenshot of the result to make the result clearer to the readers. Perhaps with a background color or border color applied. Especially because they are trying to learn this new layout module. 

See [Repl.it](https://repl.it/) for interactive code snippets.

Replicate this for further code examples.